source/prerequisites/postgres.txt

```diff
@@ -1,13 +1,87 @@
 .. _rm-prereq-postgres:
 
-================================
-Postgres Migration Prerequisites
-================================
+==================================
+PostgreSQL Migration Prerequisites
+==================================
 
 .. contents:: On this page
    :local:
    :backlinks: none
    :depth: 1
    :class: singlecol
 
-Postgres.
\ No newline at end of file
+About this Task
+---------------
+
+Connecting Relational Migrator to PostgreSQL requires
+`logical decoding 
+<https://www.postgresql.org/docs/current/logicaldecoding-explanation.html>`__ 
+enabled on your PostGreSQL instance. When you set up a new connection, 
+Relational Migrator automatically generates the required SQL code to 
+configure PostgreSQL. This document explains the purpose of those scripts.
```

Comment:
[Suggestion]

I feel like this makes it seem like the procedure is about explaining the scripts but if I understand correctly it's configuring PostgreSQL and along the way the purpose of those scripts is explained? If so, maybe we can reword this to explain that by doing this procedure you also understand the scripts that are produced.